paddle/fluid/operators/elementwise/elementwise_pow_op.h

```diff
@@ -22,15 +22,22 @@ namespace operators {
 template <typename T>
 struct PowFunctor {
   inline HOSTDEVICE T operator()(T a, T b) const {
-#ifdef __CUDA_ARCH__
-    // On CUDAPlace, std::pow(3, 1) calls pow(float, float), and
-    // it will return a float number like 2.99... , which floor to 2
-    // when cast to int by default and it is wrong.
-    // Use llrint to cast it to the nearest integer, which is 3.
+    // TODO(joejiong) in order to support functionality we comment this block
```

Comment:
1. Comment should be detailed. What functionality are you going to support? Because we should let future software engineer know your goal.

2. why is the line break at line 26?